src/sentry/south_migrations/0500_auto__del_eventmapping__del_unique_eventmapping_project_id_event_id.py

```diff
@@ -0,0 +1,1472 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = True
```

Comment:
This is good. With this, it's safe to deploy. Manually, we'll rename the table first. That allows us to surface any uncaught references. Let it sit renamed for a bit, then remove later, since there's no real rush to actually drop it.